packages/tldraw/src/translations/pt-br.json

```
@@ -83,5 +97,28 @@
   "forward": "Avançar",
   "backward": "Recuar",
   "back": "Voltar",
-  "language": "Idioma"
+  "language": "Idioma",
+  "image": "Imagem",
+  "keyboard.shortcuts": "Atalhos de Teclado",
+  "dock.position": "Posição de Ferramentas",
+  "bottom": "Inferior",
+  "top": "Superior",
+  "right": "Direita",
+  "left": "Esquerda",
+  "align.left": "Alinhar Esquerda",
+  "align.center.x": "Alinhar Centro Horizontal",
```

Sounds weird when done a literal translation

Suggestion:
```suggestion
  "align.center.x": "Alinhar ao centro na horizontal",
```